lib/ansible/module_utils/ldap.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2017-2018 Keller Fuchs (@kellerfuchs) <kellerfuchs@hashbang.sh>
+
+# This file is part of Ansible by Red Hat
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+import traceback
+from ansible.module_utils._text import to_native
+
+try:
+    import ldap
+    import ldap.sasl
+    HAS_LDAP = True
+
```

Comment:
I would move this blank line above the `HAS_LDAP`.

Reply:
What's the rationale for it?

Reply:
A lot of Python code typically has a blank line after imports.

Reply:
Exactly.  The new line should be right after the import and not after the global variable.